src/wc/wc.rs

```diff
@@ -185,6 +197,17 @@ fn wc(files: Vec<String>, settings: &Settings) -> StdResult<(), i32> {
                 }
             }
             char_count += current_char_count;
+            if shortest_line_length_initialized == false {
```

Comment:
You can probably get rid of this variable by just setting `shortest_line_length` to `usize::max()` instead of 0 and then setting `shortest_line_length` to 0 outside the loop if `line_count == 0`.